openedx/frontend-app-course-authoring
# feat: add checklist page

JIRA Ticket: [TNL-10755](https://2u-internal.atlassian.net/browse/TNL-10755)
> Copy over to course-authoring:
>
> - CourseChecklist
> - CourseChecklistPage
> - CourseHealthCheckIndex
>
> AC:
> Page should render as expected. The render generally same as the page in Studio, just under a different URL.

This PR adds the code for the Checklist page. This additional support `studio-frontend` [DEPR-381](https://github.com/openedx/studio-frontend/issues/381). 

Testing
1. Open the legacy checklist page
2. In a new window open the mfe checklist page
3. Check that all of the content matches
4. In the MFE page test that all of the update buttons and links lead to the correct location
